README.md

```diff
@@ -131,13 +134,16 @@ Or use one of the domain definitions included with `lagtraj` (e.g.
 ```bash
 $> python -m lagtraj.domain.download lagtraj://eurec4a_circle <start_date> <end_date>
 ```
-the `<start_date>` and `<end_date>` should be formatted as `YYYY/MM/DD`, e.g. `2020/02/01` for the 2nd of February 2020.
+the `<start_date>` and `<end_date>` should be formatted as `YYYY/MM/DD`, e.g. `2020/02/02` for the 2nd of February 2020.
 
 An optional flag `--retry-rate <num_minutes>` causes `lagtraj` to continue
 retrying download of submitted data requests every `num_minutes` minutes until
 all data has been downloaded. Every time this command is run it will attempt to
 download only data not yet downloaded.
 
+You can monitor the status of your requests via the [CDS requests page](ihttps://cds.climate.copernicus.eu).
```

Comment:
I think you have a superfluous `i` at the beginning of the link, should be `https://` I think